### DISCUSSION TITLE
Restore a deleted GitHub account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23446] [New to GitHub]
<p>How to restore a deleted GitHub account?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you may try requesting it but there is no assurance</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/contact/account">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181077133-34713b0b-af26-48bf-a077-64d68dbb639a.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/contact/account" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/contact/account" target="_blank" rel="noopener">Accounts - GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Reply by Niranjan963764** [reply]
Hi there

**Reply by ghost** [reply]
giúp tôi khôi phục tài khoản đã xóa với ạ

**Comment by khaja0500** [comment]
<a href="https://support.github.com/contact/account" target="_blank" rel="noopener">support.github.com</a>
